### PR TITLE
Initial setup of repository

### PR DESCRIPTION
**Pull request by philippzagar**
# Initial setup of repository

## :recycle: Current situation & Problem
Currently, the repo is mostly empty except for a template package.

## :gear: Release Notes 
- Initial setup of the `SpeziChat` repo which previously was a target in the [SpeziML](https://github.com/StanfordSpezi/SpeziML) repo
- Improved README and documentation via DocC

## :books: Documentation
--

## :white_check_mark: Testing
UI Tests are included

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md):
- [X] I agree to follow the [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by philippzagar** [comment]
@PSchmiedmayer Same as in https://github.com/StanfordSpezi/SpeziSpeech/pull/2, I think what's left here is a proper DOI as well as the push to the Swift Package Index (the latter one can probably only be done after merging to `main`, the links to the index are already set correctly, that's thy the Markdown Link Check fails).

File: .github/workflows/build-and-test.yml
Comment (PSchmiedmayer) [inline .github/workflows/build-and-test.yml:1]: All the same comments from https://github.com/StanfordSpezi/SpeziSpeech/pull/2/files#diff-bc668a2c9f2299cef15b222055b4b4d5311646caec2e7610e540cee18ae9b948 also apply to this file.

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -22,4 +22,3 @@ jobs:
   markdown_link_check:
     name: Markdown Link Check
     uses: StanfordBDHG/.github/.github/workflows/markdown-link-check.yml@v2
-
```
Comment (PSchmiedmayer) [inline .github/workflows/pull_request.yml:25]: This diff is not necessary.

File: CITATION.cff
Comment (PSchmiedmayer) [inline CITATION.cff:1]: Same comments as in https://github.com/StanfordSpezi/SpeziSpeech/pull/2/files#diff-9b9f76394a441ff40bf8d7a0f3e0ddd7ae97abfa9a9a6abaedccf76dc4d51295.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -14,3 +14,6 @@ SpeziChat contributors
 ====================
 
 * [Paul Schmiedmayer](https://github.com/PSchmiedmayer)
+* [Vishnu Ravi](https://github.com/vishnuravi)
+* [Philipp Zagar](https://github.com/philippzagar)
+* [Adrit Rao](https://github.com/AdritRao)
```
Comment (PSchmiedmayer) [inline CONTRIBUTORS.md:19]: 🎉🚀

File: Package.swift [hunk 3]
```diff
@@ -21,16 +21,13 @@ let package = Package(
         .library(name: "SpeziChat", targets: ["SpeziChat"])
     ],
     dependencies: [
-        .package(url: "https://github.com/StanfordSpezi/Spezi", .upToNextMinor(from: "0.8.0"))
+        .package(url: "https://github.com/StanfordSpezi/SpeziSpeech", branch: "feat/init-setup")    // .upToNextMinor(from: "0.1.0")
```
Comment (PSchmiedmayer) [inline Package.swift:24]: TODO to fix this once the Speech PR is merged.

File: README.md
Comment (PSchmiedmayer) [inline README.md:1]: The same comments as the ones in https://github.com/StanfordSpezi/SpeziSpeech/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 apply to this README as well 👍

File: README.md [hunk 4]
```diff
@@ -8,72 +8,131 @@ SPDX-License-Identifier: MIT
...
 -->
 
-# SpeziChat
+# Spezi Chat
 
 [![Build and Test](https://github.com/StanfordSpezi/SpeziChat/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/StanfordSpezi/SpeziChat/actions/workflows/build-and-test.yml)
 [![codecov](https://codecov.io/gh/StanfordSpezi/SpeziChat/graph/badge.svg?token=b2Dn0r9eo6)](https://codecov.io/gh/StanfordSpezi/SpeziChat)
+[![DOI](https://zenodo.org/badge/573230182.svg)](https://zenodo.org/badge/latestdoi/573230182)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2FStanfordSpezi%2FSpeziChat%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/StanfordSpezi/SpeziChat)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2FStanfordSpezi%2FSpeziChat%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/StanfordSpezi/SpeziChat)
```
Comment (PSchmiedmayer) [inline README.md:17]: The same comments as the ones 
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -8,72 +8,131 @@ SPDX-License-Identifier: MIT
...
 
+|![Screenshot displaying the regular chat view.](Sources/SpeziChat/SpeziChat.docc/Resources/ChatView.png#gh-light-mode-only) ![Screenshot displaying the regular chat view.](Sources/SpeziChat/SpeziChat.docc/Resources/ChatView~dark.png#gh-dark-mode-only)|![Screenshot displaying the text input chat view.](Sources/SpeziChat/SpeziChat.docc/Resources/ChatView+TextInput.png#gh-light-mode-only) ![Screenshot displaying the text input chat view.](Sources/SpeziChat/SpeziChat.docc/Resources/ChatView+TextInput~dark.png#gh-dark-mode-only)|![Screenshot displaying the voice input chat view.](Sources/SpeziChat/SpeziChat.docc/Resources/ChatView+VoiceInput.png#gh-light-mode-only) ![Screenshot displaying the voice input chat view.](Sources/SpeziChat/SpeziChat.docc/Resources/ChatView+VoiceInput~dark.png#gh-dark-mode-only)
+|:--:|:--:|:--:|
+|[`ChatView`](https://swiftpackageindex.com/stanfordspezi/spezichat/documentation/spezichat/chatview)|[`ChatView`](https://swiftpackageindex.com/stanfordspezi/spezichat/documentation/spezichat/chatview)|[`ChatView`](https://swiftpackageindex.com/stanfordspezi/spezichat/documentation/spezichat/chatview)|
```
Comment (PSchmiedmayer) [inline README.md:27]: Thank you for all the screenshots, very nice!!

File: README.md [hunk 6]
```diff
@@ -8,72 +8,131 @@ SPDX-License-Identifier: MIT
...
 > [!IMPORTANT]  
-> If your application is not yet configured to use Spezi, follow the [Spezi setup article](https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/initial-setup) setup the core Spezi infrastructure.
+> If your application is not yet configured to use Spezi, follow the [Spezi setup article](https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/initial-setup) to setup the core Spezi infrastructure.
+   
+## Examples
 
+### Chat View
 
-### 2. Register the Module
+The [`ChatView`](https://swiftpackageindex.com/stanfordspezi/spezichat/documentation/spezichat/chatview) provides a basic reusable chat view which includes a message input field. The input can be either typed out via the iOS keyboard or provided as voice input and transcribed into written text.
```
Comment (PSchmiedmayer) [inline README.md:44]: Thank you for all the examples in here; great job!!

File: Sources/SpeziChat/SpeziChat.docc/SpeziChat.md
Comment (PSchmiedmayer) [inline Sources/SpeziChat/SpeziChat.docc/SpeziChat.md:1]: Thank you for the nice DocC article here!!

File: Tests/UITests/TestApp/ChatTestView.swift
Comment (PSchmiedmayer) [inline Tests/UITests/TestApp/ChatTestView.swift:1]: Nice to see the example implementation here! 🚀

**Review by PSchmiedmayer — CHANGES_REQUESTED** [review]
Thank you for adding the code to this repo from Spezi ML and adding all the documentation surrounding it!

Similar to the Speech PR; I had a few comments that would be great to be addressed before we merge the PR.

The overall PR looks very great and all the documentation added is more than appreciated! 🚀

**Comment by philippzagar** [comment]
@PSchmiedmayer PR is ready for another round of review. The only left TODO will be incorporated once SpeziSpeech is merged and tagged.

**Review by PSchmiedmayer — APPROVED** [review]
Thank you @philippzagar; this looks great!

**Comment by philippzagar** [comment]
@PSchmiedmayer Similar to https://github.com/StanfordSpezi/SpeziSpeech, please merge the PR and tag a release :)

**Comment by PSchmiedmayer** [comment]
Thank you for all the setup here @philippzagar! I have merged the PR, tagged a release, and it should also start the process of creating a DOI for the repo.

Feel free to create a similar PR to Spezi Speech once all the processes are finished. Thank you for all the work in setting these things up!

**Comment by philippzagar** [comment]
Thanks @PSchmiedmayer, will add the DOI as well as SPI things soon!

**Pull request merged**